server/testkit/tests/testkit/https_extractors.rs

```diff
@@ -5,12 +5,13 @@ use std::{
 
```

Comment:
Where are the tests in this PR for the proxy v2 headers or `HttpAddressInfo::XForwardForAllSourcesTrusted`?